src/components/UI/GlobalStyles.js

```diff
@@ -1,22 +1,27 @@
 import { createGlobalStyle } from 'styled-components';
 
-const GlobalStyle = createGlobalStyle`
+const GlobalStyles = createGlobalStyle`
 * {
     box-sizing: border-box;
     margin: 0;
     padding: 0;
   }
 
-  html {
+html {
   font-size: 62.5%;
```

Comment:
Our head couch told us to do so, so that it is easier to work with rem.
But since I want to pay special attention to accessibility, I am open to changing that if it does affect accessibility.

Reply:
Declaring font-size inside the `html` selector doesn't affect `rem`, as `rem` would usually be declared inside the `:root` selector, so I'm not entirely sure how the two are related. In this instance, rem and % would do pretty much the same thing as part of the `html` selector. For example: setting the font-size as 1.5rem would be equal to setting the font-size to 150%. The major difference would be that while rem stays relative to the root em declared within the `:root` selector, percentages behave similar to em in how they get inherited and scaled up/down according to their parent element. I'm not really seeing an issue with using percentage as part of the `html` selector, I just figured I'd ask. :)